src/Stats/index.jsx

```diff
@@ -1,33 +1,103 @@
-import React from 'react';
-import { View, ScrollView } from 'react-native';
-import { ListItem } from 'react-native-elements';
+import React, { useState } from 'react';
+import { View, ScrollView, Text } from 'react-native';
+import { ListItem, Overlay, Button } from 'react-native-elements';
 import { useSelector } from 'react-redux';
-import moment from 'moment';
-import utils from './utils';
 import styles from '../Common/themeStyle';
+import MainHeader from '../Common/MainHeader';
+import LargeButton from '../Common/LargeBtn';
+import GradientIcon from '../Common/icon';
+import utils from './utils';
+import SubHeader from '../Common/SubHeader';
+
+const moment = require('moment');
 
 export default function Stats() {
   const { transactions } = useSelector((state) => state.transactions);
+  const [currentMonthSet, setCurrentMonthSet] = useState(utils.getDateSet(moment(), 'months'));
+  const [currentYearSet, setCurrentYearSet] = useState(utils.getDateSet(moment(), 'years'));
```

Comment:
i think you don really need two variables for these 'sets', it is enough if you have a `timePeriodOptions` or something better name and you can update it every time the current view changes

Reply:
Done